sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -0,0 +1,113 @@
+    serviceVersion?: SUPPORTED_API_VERSIONS;
+}
+
+// @public
+export interface CreateRoleAssignmentOptions extends coreHttp.OperationOptions {
+}
+
+// @public
+export interface DeleteRoleAssignmentOptions extends coreHttp.OperationOptions {
+}
+
+// @public
+export interface GetRoleAssignmentOptions extends coreHttp.OperationOptions {
+}
+
+// @public
+export class KeyVaultAccessControlClient {
+    constructor(vaultUrl: string, credential: TokenCredential, pipelineOptions?: AccessControlClientOptions);
+    createRoleAssignment(roleScope: RoleAssignmentScope, name: string, roleDefinitionId: string, principalId: string, options?: CreateRoleAssignmentOptions): Promise<KeyVaultRoleAssignment>;
+    deleteRoleAssignment(roleScope: RoleAssignmentScope, name: string, options?: DeleteRoleAssignmentOptions): Promise<KeyVaultRoleAssignment>;
+    getRoleAssignment(roleScope: RoleAssignmentScope, name: string, options?: GetRoleAssignmentOptions): Promise<KeyVaultRoleAssignment>;
+    listRoleAssignments(roleScope: RoleAssignmentScope, options?: ListRoleAssignmentsOptions): PagedAsyncIterableIterator<KeyVaultRoleAssignment>;
+    listRoleDefinitions(roleScope: RoleAssignmentScope, options?: ListRoleDefinitionsOptions): PagedAsyncIterableIterator<KeyVaultRoleDefinition>;
+    readonly vaultUrl: string;
+}
+
+// @public
+export interface KeyVaultPermission {
+    actions?: string[];
+    dataActions?: string[];
+    notActions?: string[];
+    notDataActions?: string[];
+}
+
+// @public
+export interface KeyVaultRoleAssignment {
+    readonly id: string;
+    readonly name: string;
+    properties: KeyVaultRoleAssignmentPropertiesWithScope;
+    readonly type: string;
+}
+
+// @public
+export interface KeyVaultRoleAssignmentProperties {
+    principalId: string;
+    roleDefinitionId: string;
+}
+
+// @public
+export interface KeyVaultRoleAssignmentPropertiesWithScope {
+    principalId: string;
+    roleDefinitionId: string;
+    scope?: RoleAssignmentScope;
+}
+
+// @public
+export interface KeyVaultRoleDefinition {
+    assignableScopes: string[];
+    description: string;
+    readonly id: string;
+    readonly name: string;
+    permissions: KeyVaultPermission[];
+    roleName: string;
+    roleType: string;
+    readonly type: string;
+}
+
+// @public
+export const LATEST_API_VERSION = "7.1";
+
+// @public
+export interface ListRoleAssignmentsOptions extends coreHttp.OperationOptions {
+}
+
+// @public
+export interface ListRoleAssignmentsPageSettings {
+    continuationToken?: string;
+}
+
+// @public
+export interface ListRoleDefinitionsOptions extends coreHttp.OperationOptions {
+}
+
+// @public
+export interface ListRoleDefinitionsPageSettings {
+    continuationToken?: string;
+}
+
+// @public
+export type RoleAssignmentScope = "/" | "/keys" | string;
+
+// @public
+export const SDK_VERSION: string;
+
+// @public
+export type SUPPORTED_API_VERSIONS = "7.0" | "7.1" | "7.2-preview";
```

Comment:
for administration isn't only 7.2-preview supported rn?

Reply:
thank you!